lib/xdrgen/generators/rust.rb

```diff
@@ -76,7 +76,7 @@ def render_top_matter(out)
           //  #{@output.relative_source_paths.join("\n//  ")}
         EOS
         out.break
-        out.puts "#![allow(clippy::missing_errors_doc, clippy::unreadable_literal)]"
+        out.puts "#![allow(clippy::missing_errors_doc, clippy::unreadable_literal, clippy::needless_question_mark)]"
```

Comment:
I did not modify the relevant code, so I am not sure why clippy is giving a warning. And I encountered some difficulties in fixing this warning, so let's add it first.